app/scripts/services/membership/roles.js

```diff
@@ -7,15 +7,15 @@ angular
     APIService,
     DataService) {
 
-    var roleBindingsVersion = APIService.getPreferredVersion('rolebindings');
-    var clusterRoleBindingsVersion = APIService.getPreferredVersion('clusterroles');
+    var rolesVersion = APIService.getPreferredVersion('roles');
```

Comment:
The only change is the top one, I just renamed the rest of the variables as they were not clear (prob led to this sliding by).